src/components/BorrowerProfile/LoanStory.vue

```diff
@@ -96,13 +131,30 @@ export default {
 						id
 						name
 					}
+					geocode {
+						country {
+							isoCode
+						}
+					}
+					tags
+					gender
+					sector {
+						id
+						name
+					}
```

Comment:
This is a prefetched query. Can these items be placed in a new query activated by the mounted hook instead?

Reply:
Looks like you've already duplicated them in mounted so maybe these can just be removed...

Reply:
Thought I removed them, thanks!